src/geoserver/epsg_properties.py

```diff
@@ -0,0 +1,91 @@
+           'AUTHORITY["EPSG","7030"]],AUTHORITY["EPSG","6326"]],PRIMEM["Greenwich",0,AUTHORITY["EPSG","8901"]],UNIT["degree",'
+           '0.0174532925199433,AUTHORITY["EPSG","9122"]],AUTHORITY["EPSG","4326"]],PROJECTION["Transverse_Mercator"],'
+           'PARAMETER["latitude_of_origin",0],PARAMETER["central_meridian",21],PARAMETER["scale_factor",0.9996],'
+           'PARAMETER["false_easting",500000],PARAMETER["false_northing",0],UNIT["metre",1,AUTHORITY["EPSG","9001"]],AXIS["Easting",'
+           'EAST],AXIS["Northing",NORTH],AUTHORITY["EPSG","32634"]]',
+    32633: '32633=PROJCS["WGS 84 / UTM zone 33N",GEOGCS["WGS 84",DATUM["WGS_1984",SPHEROID["WGS 84",6378137,298.257223563,'
+           'AUTHORITY["EPSG","7030"]],AUTHORITY["EPSG","6326"]],PRIMEM["Greenwich",0,AUTHORITY["EPSG","8901"]],UNIT["degree",'
+           '0.0174532925199433,AUTHORITY["EPSG","9122"]],AUTHORITY["EPSG","4326"]],PROJECTION["Transverse_Mercator"],'
+           'PARAMETER["latitude_of_origin",0],PARAMETER["central_meridian",15],PARAMETER["scale_factor",0.9996],'
+           'PARAMETER["false_easting",500000],PARAMETER["false_northing",0],UNIT["metre",1,AUTHORITY["EPSG","9001"]],AXIS["Easting",'
+           'EAST],AXIS["Northing",NORTH],AUTHORITY["EPSG","32633"]]',
+    102067: '102067=PROJCS["S-JTSK_Krovak_East_North",GEOGCS["GCS_S_JTSK",DATUM["Jednotne_Trigonometricke_Site_Katastralni",'
+            'SPHEROID["Bessel_1841",6377397.155,299.1528128]],PRIMEM["Greenwich",0],UNIT["Degree",0.017453292519943295]],'
+            'PROJECTION["Krovak"],PARAMETER["False_Easting",0],PARAMETER["False_Northing",0],PARAMETER["Pseudo_Standard_Parallel_1",'
+            '78.5],PARAMETER["Scale_Factor",0.9999],PARAMETER["Azimuth",30.28813975277778],PARAMETER["Longitude_Of_Center",'
+            '24.83333333333333],PARAMETER["Latitude_Of_Center",49.5],PARAMETER["X_Scale",-1],PARAMETER["Y_Scale",1],'
+            'PARAMETER["XY_Plane_Rotation",90],UNIT["Meter",1],AUTHORITY["EPSG","102067"]]',
+}
+
+
+def get_srs_list_from_epsg_file(file_path):
+    result = set()
+    if os.path.exists(file_path):
+        with open(file_path, 'r') as f:
+            epsg_props = f.read()
+        epsg_defs = re.finditer(r'^([1-9][0-9]+)=', epsg_props, re.MULTILINE)
+        result = {int(epsg_def.group(1)) for epsg_def in epsg_defs}
+    return result
+
+
+def setup_epsg(data_dir, srs_list):
+    logger.info(f"Ensuring EPSG definition for SRS in list: '{srs_list}'")
+    file_name = 'epsg.properties'
+
+    file_path = os.path.join(data_dir, 'user_projections', file_name)
+    pathlib.Path(os.path.dirname(file_path)).mkdir(parents=True, exist_ok=True)
+    old_srs_list = get_srs_list_from_epsg_file(file_path)
+    missing_srs = srs_list.difference(old_srs_list)
+    new_epsg = {code: definition for code, definition in EPSG_PROPERTIES_DEFAULT.items()
+                if code in missing_srs}
+
+    srs_to_download = missing_srs.difference(new_epsg.keys())
```

Comment:
If there is no change, write single comment, otherwise write also three comments below.